packages/sitecore-jss/src/native-fetcher.ts

```diff
@@ -182,6 +182,7 @@ export class NativeDataFetcher {
     if (!init.method) {
       init.method = init.body ? 'POST' : 'GET';
     }
+    headers.set('User-Agent', 'Sitecore JSS');
```

Comment:
using the static text here without version.
We can only get version from packageJSON, but importing it conficts with ESM build output and would fail jss-angular package build.